src/main/java/com/stripe/model/Invoice.java

```diff
@@ -25,12 +26,14 @@
 	Long date;
 	String description;
 	Discount discount;
+	Long dueDate;
```

Comment:
I believe the Java bindings only require you to add the fields on the model, not the parameters accepted to API calls. `days_until_due` is a parameter that can be passed on create, but is not a field on the invoice model.